### PR TITLE
[Feature]: Judge View

### PR DESCRIPTION
**Pull request by mkanoria**
- Add judge view to screens across the app
  - disable settings from navbar
  - update awards and performances page to show data specific to judge
  - disable adding or editing performances, deleting awards 
- Fix small dance title bug in the awards page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "@babel/runtime": "^7.13.10",
```
Comment (mkanoria) [inline package.json:18]: The project was not compiling for me locally without adding this, but can discuss if this was an issue only on my setup

File: pages/api/schools/collect.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ export default async (req, res) => {
   const session = await getSession({ req });
 
   // If session exists (thus, user is authenticated)
-  if (session && session.role === 'ADMIN') {
+  if (session && (session.role === 'ADMIN' || session.role === 'JUDGE')) {
```
Comment (mkanoria) [inline pages/api/schools/collect.js:9]: Without this the API returns a `401` for judge roles - we should return the data but filter it from the user on the FE

File: pages/api/user/collect.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ export default async (req, res) => {
   const session = await getSession({ req });
 
   // If session exists (thus, user is authenticated)
-  if (session && session.role === 'ADMIN') {
+  if (session && (session.role === 'ADMIN' || session.role === 'JUDGE')) {
```
Comment (mkanoria) [inline pages/api/user/collect.js:9]: Same as above

File: package.json (hunk 1 above)
Comment (thepenguinco) [reply package.json:18]: I guess this is fine.  Very strange behaviour though.

File: pages/api/schools/collect.js (hunk 2 above)
Comment (thepenguinco) [reply pages/api/schools/collect.js:9]: Why do the judges need to be able to call api/schools/collect, they never need to see all the schools (just the ones associated with the appropriate performances).

File: pages/api/user/collect.js (hunk 3 above)
Comment (thepenguinco) [reply pages/api/user/collect.js:9]: Why do the judges need to be able to call api/user/collect?  They never need to access the settings page.

**Review by thepenguinco — APPROVED** [review]
Few comments.  Otherwise LGTM.

File: pages/api/schools/collect.js (hunk 2 above)
Comment (mkanoria) [reply pages/api/schools/collect.js:9]: Looked into it, this is used to populate the filter options for the [performances page](https://github.com/uwblueprint/dancefest-web/blob/staging/pages/performances/index.js#L114)

File: pages/api/user/collect.js (hunk 3 above)
Comment (mkanoria) [reply pages/api/user/collect.js:9]: This is to populate the 'judges' field of each event card iirc in [index.js](https://github.com/uwblueprint/dancefest-web/blob/staging/pages/performances/index.js#L114)

**Comment by thepenguinco** [comment]
Okay got it, that's fine then.

**Pull request merged**